modules/serverless-rn-1-4-0.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-4-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.4.0
+
+[id="new-features-1-4-0_{context}"]
+== New features
+* {ServerlessProductName} 1.4.0 is available on {product-title} 4.2 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.11.1.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.11.0.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.11.1.
+* Knative Client is now available for download through the *Command Line Tools* page in the {product-title} web console.
+* The `KnativeServing` object's API group has changed in this release from `serving.knative.dev` to `operator.knative.dev`. You will need to adjust any of your scripts or applications that rely on the old API group to use the new API group. The {ServerlessProductName} installation instructions have been updated to use the new API group.
++
+When upgrading from {ServerlessProductName} 1.3.0 to 1.4.0, the {ServerlessOperatorName} will create a `KnativeServing` custom resource (CR) in the new API group for you. This CR will be a mirror of the `KnativeServing` CR in the old group that was used in {ServerlessProductName} 1.3.0.
++
+If you need to keep using the old group temporarily, you can use the old CR as before. However, this CR is deprecated and will eventually be removed.
++
+Once you have updated references to the new API group, you can remove any older CR versions and use the newly deployed `KnativeServing` CR instead.
```

Comment:
If the user removes the old CR it will uninstall the whole Serverless. IMO, we need to put the text from the note below directly to this paragraph so that nobody can miss that.